lib/org/chromium/webidl/ast/Definition.js

```diff
@@ -14,6 +14,10 @@ foam.CLASS({
       of: 'org.chromium.webidl.ast.DefinitionData',
       name: 'definition',
     },
+    {
+      name: 'id',
+      factory: function() { return this.definition.id; },
```

Comment:
Here, let's delegate to this.getName(), which delegates to definition.getName(), and so on. That way other AST nodes can implement getName(), without needing to have an id.